### DISCUSSION TITLE
Corrupted Owl

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2783] [Ideas]
Corrupted Owl is a corrupted version of the owl. It looks unpleasant. Its mouth is open, its color is purple, and its eyes are light purple and one of its eyes has a root in the eye.

He carries a ball of roots on his paws that when thrown at the player, explodes with small roots just like corrupted igel when it hits a wall while rolling.

Here's a look at what Corrupted Owl looks like

![corrupted_owl](https://github.com/SuperTux/supertux/assets/139823928/d82d89f4-f3af-4d81-84aa-9198d60bb450)

I don't know if this idea is going to be added. But if anyone sees this idea, give me your thoughts on my idea

**Comment by retcinder** [comment]
Not to sound like a certain floating triangle buuuuuuuuut this seems unnecessary. The regular owl could very easily do something like this.

**Reply by tylerandari13** [reply]
To add to this in the story the zeeklings and owls fly away which is why you dont see them in the corrupted forest.

**Reply by Linusintro** [reply]
If the badguys get corrupted by roots then owls and zeeklings can't be corrupted, because they don't touch the ground.

**Reply by Rusty-Box** [reply]
Exactly! That was part of my story explaination for no Corupted Owl and Zeekling being present. They flew away to avoid getting corrupted in the first place. Like how birds fly away to flee the scene.
